tests/test_fits_to_zarr.py

```
@@ -516,6 +536,60 @@ def test_rechunk_lm_for_zarr_chunk_lm_zero_single_spatial_chunk():
     assert out["SKY"].data.chunks == ((100,), (100,))
 
 
+def test_rechunk_lm_for_zarr_fixes_irregular_wcs_header_str_chunks(tmp_path):
+    """Non-uniform dask chunks on aux vars (e.g. wcs along frequency) must be Zarr-safe."""
+    import dask.array as da
+    import numpy as np
+    import xarray as xr
+
+    mod = _import_module()
+    n = 4
+    l = np.linspace(-1.0, 1.0, 32)
```

Ruff (0.15.12)

[error] Ambiguous variable name: `l`

(E741)